examples/examples.py

```diff
@@ -0,0 +1,54 @@
+# %%
+import logging
+import os
+from os import getenv
+
+from assume import World, load_scenario_folder
+
+log = logging.getLogger(__name__)
+
+os.makedirs("./examples/outputs", exist_ok=True)
+EXPORT_CSV_PATH = str(getenv("EXPORT_CSV_PATH", "./examples/outputs"))
+
+os.makedirs("./examples/local_db", exist_ok=True)
+
+availabe_examples = {
+    "small": {"scenario": "example_01a", "study_case": "example_01a"},
```

Comment:
shouldn't we just say `small` instead of `example_01a`. Same for all the other scenarios..?